Lib/slapdtest/_slapdtest.py

```diff
@@ -596,3 +623,43 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         cls.server.stop()
+
+
+class PPolicyEnabledSlapdObject(SlapdObject):
```

Comment:
Do you think this class will be needed for more than just one test? If not, would it be better to define the class just in the file for that test?